<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LaserMainWindow</class>
 <widget class="QMainWindow" name="LaserMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1478</width>
    <height>1046</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>780</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_print_list">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>120</y>
      <width>741</width>
      <height>881</height>
     </rect>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>3</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>80</number>
    </attribute>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>序号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>原序号</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignVCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>状态</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>名称</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>12</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>已打印</string>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>THYJJJ12121KKKwewewK</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_edit_item">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>850</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>编辑信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_create_table">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>850</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>生成报表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_exit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>910</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>680</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>结束</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>操作员</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>680</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>580</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>打印条目</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_select_all">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>全选</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_select_unprint">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>70</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>未打印</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_select_invert">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>反选</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_select_items">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>580</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>空</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_select_excel_file">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_deal_excel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>500</y>
      <width>381</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>导入文件</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>制作 by kobe</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textEdit_product_file">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>390</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>客户.xls</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textEdit_operator_name">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>290</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_custom_file">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>891</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ff0000;&quot;&gt;客户文件信息:空&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textEdit_begin_num">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>680</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textEdit_end_num">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>680</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_filter_num">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>670</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>选择</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_total_num">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>640</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>总条数目： 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>客户名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textEdit_custom_name">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_print">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="format">
     <string>进度: 0/0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stop">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_select_custom">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>330</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>空</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_select_operator">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>290</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>空</string>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="tableWidget_combo_select">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>170</y>
      <width>241</width>
      <height>831</height>
     </rect>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>3</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>80</number>
    </attribute>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>序号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>名称</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>120</y>
      <width>16</width>
      <height>881</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_group_print">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>110</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>打印组</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="textEdit_output_path">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>430</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>C:\</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_select_output_path">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>导出路径</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textEdit_stop_index">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>190</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_exit</sender>
   <signal>clicked()</signal>
   <receiver>LaserMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>852</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>953</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>kobe_show_single()</signal>
  <slot>on_pushButton_start_clicked()</slot>
  <slot>on_pushButton_mix_table_clicked()</slot>
  <slot>on_pushButton_create_table_clicked()</slot>
  <slot>on_comboBox_start_index_currentIndexChanged()</slot>
  <slot>on_comboBox_end_index_currentIndexChanged()</slot>
  <slot>on_checkBox_select_all_stateChanged()</slot>
  <slot>on_checkBox_select_unprint_stateChanged()</slot>
  <slot>on_comboBox_select_items_objectNameChanged()</slot>
  <slot>on_comboBox_select_items_currentIndexChanged()</slot>
  <slot>on_pushButton_select_excel_file_clicked()</slot>
  <slot>on_checkBox_select_invert_stateChanged()</slot>
  <slot>on_pushButton_deal_excel_clicked()</slot>
 </slots>
</ui>
